src/shared/utils/__tests__/utils.test.ts

```diff
@@ -289,92 +180,44 @@ describe("Util Functions", () => {
-          privacy: {
-            obfuscated: [],
-          },
-        },
-      };
-      expect(utils.getTargetHash(document)).toStrictEqual(
-        "6e3b3b131db956263d142f42a840962d31359fff61c28937d9d1add0ca04c89e"
+      expect(utils.getTargetHash(v3WrappedVerifiableDocument)).toStrictEqual(
+        "797faecb7662eca4e73a82c25a2387d57acd666354c228291fcd664e6aaeb1ba"
       );
     });
     test("should return error when document is not OpenAttestation document", async () => {
       const document: any = {
         signature: {
           type: "SHA3MerkleProof",
-          targetHash: "64b2ed566455d0adbc798a8f824f163d87276dcbd66cacff8a6a4ba28fb800fc",
+          targetHash: "123",
```

Comment:
```suggestion
          targetHash: "c5d53262962b192c5c977f2252acd4862f41cc1ccce7e87c5b406905a2726692",
```

sorry lets use back a proper looking hash, the `123` makes me think that incorrect hash matters to return the error for this test, whereas is testing for wrong object shape

Reply:
ok